### PR TITLE
Feat/i2 p6 post api

### PR DESCRIPTION
**Pull request by alexjihun8**
Created GET, POST DELETE APIs for 'posts' data model.

### CODE REVIEW

File: backend/posts/views.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions, viewsets
+
+from .models import Post
+from .serializers import PostSerializer
+
+
+class PostViewSet(viewsets.ModelViewSet):
```
Comment (OctoHoon) [inline backend/posts/views.py:7]: Nice Work!

File: backend/posts/admin.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class PostAdmin(admin.ModelAdmin):
+    list_display = ['id', 'user', 'restaurant_id', 'title', 'rating', 'created_at']
+    search_fields = ['title', 'description']
+    list_filter = ['created_at', 'rating']
+    ordering = ['-created_at']
+
+@admin.register(PostPhoto)
+class PostPhotoAdmin(admin.ModelAdmin):
+    list_display = ['id', 'post', 'photo_url']
+    search_fields = ['photo_url']
```
Comment (khinwaiyan) [inline backend/posts/admin.py:16]: admin

File: backend/README.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
 # Install Dependencies
 
-Install poetry:
+First, the project requires python version above 3.11. If not, install pyenv:
+
+`curl https://pyenv.run | bash`
+
+After installation, check the log message and follow the instruction to add pyenv to your shell. (Modify `~/.bashrc`)
+
+Then, install python 3.11.0:
```
Comment (hzlcodus) [inline backend/README.md:9]: 친절한 리드미 감사합니다 ^_^

**Review by hzlcodus — APPROVED** [review]

File: backend/README.md (hunk 3 above)
Comment (dlwocks31) [reply backend/README.md:9]: 감사합니다!

File: backend/posts/admin.py (hunk 2 above)
Comment (dlwocks31) [reply backend/posts/admin.py:16]: cool

File: backend/posts/views.py (hunk 1 above)
Comment (dlwocks31) [reply backend/posts/views.py:7]: Thanks!

**Pull request merged**